iavl/iavl_proof2.go

```diff
@@ -58,27 +60,28 @@ func (p *PathToKey) isRightmost() bool {
 	return true
 }
 
-type innerPathPair struct {
-	left  []IAVLProofInnerNode
-	right []IAVLProofInnerNode
+func (p *PathToKey) pop() *PathToKey {
```

Comment:
Ah you're right - in fact it used to, but I changed its implementation and didn't think to change its name.